browser/ui/webui/settings/brave_import_data_handler.cc

```diff
@@ -115,14 +109,24 @@ void BraveImportDataHandler::NotifyImportProgress(
   }
 }
 
+void BraveImportDataHandler::HandleImportData(const base::Value::List& args) {
+  ImportDataHandler::HandleImportData(args);
+}
```

Comment:
As this just calls base class method, I think this could be deleted.
and we can add `BraveImportBulkDataHandler` as a friend of `ImportDataHandler`.